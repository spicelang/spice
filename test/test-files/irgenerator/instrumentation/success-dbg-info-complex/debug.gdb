# Do not print non-reproducible information
set print address off
set print thread-events off
set print inferior-events off

# Preparation
break source.spice:42
run

# Runtime
info locals
print vi
print vi.size
continue

# Quit
quit