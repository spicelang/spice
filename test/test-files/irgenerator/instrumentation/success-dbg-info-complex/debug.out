GNU gdb (Ubuntu 15.0.50.20240403-0ubuntu1) 15.0.50.20240403-git
Copyright (C) 2024 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>
This is free software: you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.
Type "show copying" and "show warranty" for details.
This GDB was configured as "x86_64-linux-gnu".
Type "show configuration" for configuration details.
For bug reporting instructions, please see:
<https://www.gnu.org/software/gdb/bugs/>.
Find the GDB manual and other documentation resources online at:
    <http://www.gnu.org/software/gdb/documentation/>.

For help, type "help".
Type "apropos word" to search for commands related to "word"...
Reading symbols from ./source...
Breakpoint 1: file ./test-files/irgenerator/instrumentation/success-dbg-info-complex/source.spice, line 42.
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib/x86_64-linux-gnu/libthread_db.so.1".

Breakpoint 1, main (_argc=1, _argv=...)
    at ./test-files/irgenerator/instrumentation/success-dbg-info-complex/source.spice:42
42	    it.next();
result = 0
vi = {contents = , capacity = 5, size = 5}
it = {vector = , cursor = 4}
pair = {first = 2, second = }
$1 = {contents = , capacity = 5, size = 5}
$2 = 5
All assertions passed!
[Inferior 1 (process 216480) exited normally]
